mongodb/intro/aggregation-iv/README.md

```diff
@@ -0,0 +1,9 @@
+name: Aggregation IV
+
+description: More accumulators and operators to use with aggregation.

```

Comment:
we should try to mention what is taught in this workout. "More accumulators and operators..." sounds pretty vague.

Maybe something like: "Pushing, summing and other accumulators."